l2tdevtools/build_helpers/dpkg.py

```diff
@@ -186,11 +190,17 @@ def _CreateOriginalSourcePackageFromZip(
             zip_info = zip_file.getinfo(filename)
             tar_info = tarfile.TarInfo(filename)
             tar_info.size = zip_info.file_size
+            date_time = zip_info.date_time
```

Comment:
IMHO it is worth a comment to what you're accomplishing here and why

Reply:
Done